helm/cluster/templates/clusterapi/controlplane/_helpers_flatcar.tpl

```diff
@@ -0,0 +1,86 @@
+{{- define "cluster.kubeadmControlPlane.kubeadmConfigSpec.ignition.containerLinuxConfig.additionalConfig.systemd.units" }}
```

Comment:
Helper template for systemd units configuration in KubeadmControlPlane.

This got updated in a later PR to include common systemd units across both CP and workers, as well as CP-specific systemd units.